From ae28cbb12fc80f62230ad06cb44ad0aaaa9e2048 Mon Sep 17 00:00:00 2001
From: Jean Delvare <jdelvare@suse.de>
Date: Tue, 29 Jan 2019 14:11:46 +0100
Subject: [PATCH 07/40] Use comments to separate sections in manual pages

Blank lines are interpreted as actual blank lines in manual pages.
Use comments instead to visually separate sections. Otherwise an
extra blank like is inserted before every section.

Signed-off-by: Jean Delvare <jdelvare@suse.de>
---
 man/biosdecode.8 | 11 ++++++++---
 man/dmidecode.8  | 18 +++++++++++-------
 man/ownership.8  | 10 +++++++---
 man/vpddecode.8  | 12 +++++++-----
 4 files changed, 33 insertions(+), 18 deletions(-)

diff --git a/man/biosdecode.8 b/man/biosdecode.8
index a96eb68..cf7d4db 100644
--- a/man/biosdecode.8
+++ b/man/biosdecode.8
@@ -1,10 +1,12 @@
 .TH BIOSDECODE 8 "February 2007" "dmidecode"
+.\"
 .SH NAME
 biosdecode \- \s-1BIOS\s0 information decoder
+.\"
 .SH SYNOPSIS
 .B biosdecode
 .RB [ OPTIONS ]
-
+.\"
 .SH DESCRIPTION
 .B biosdecode
 parses the \s-1BIOS\s0 memory and prints information about all structures (or
@@ -54,7 +56,7 @@ started its life as a part of
 .B dmidecode
 but as more entry point types were added, it was moved to a different
 program.
-
+.\"
 .SH OPTIONS
 .TP
 .BR "-d" ", " "--dev-mem FILE"
@@ -68,17 +70,20 @@ Display usage information and exit
 .TP
 .BR "-V" ", " "--version"
 Display the version and exit
-
+.\"
 .SH FILES
 .I /dev/mem
+.\"
 .SH BUGS
 Most of the time,
 .B biosdecode
 prints too much information (you don't really care about addresses)
 or not enough (because it doesn't follow pointers and has no lookup
 tables).
+.\"
 .SH AUTHORS
 Alan Cox, Jean Delvare
+.\"
 .SH "SEE ALSO"
 .BR dmidecode (8),
 .BR mem (4),
diff --git a/man/dmidecode.8 b/man/dmidecode.8
index 52100a8..1f6529d 100644
--- a/man/dmidecode.8
+++ b/man/dmidecode.8
@@ -1,10 +1,12 @@
 .TH DMIDECODE 8 "January 2019" "dmidecode"
+.\"
 .SH NAME
 dmidecode \- \s-1DMI\s0 table decoder
+.\"
 .SH SYNOPSIS
 .B dmidecode
 .RB [ OPTIONS ]
-
+.\"
 .SH DESCRIPTION
 .B dmidecode
 is a tool for dumping a computer's \s-1DMI\s0 (some say \s-1SMBIOS\s0) table
@@ -58,7 +60,7 @@ displayed value.
 Decoded values. The information presented of course depends on the type
 of record. Here, we learn about the board's manufacturer, model, version
 and serial number.
-
+.\"
 .SH OPTIONS
 .TP
 .BR "-d" ", " "--dev-mem FILE"
@@ -158,10 +160,9 @@ is run on a system with BIOS that boasts new SMBIOS specification, which
 is not supported by the tool yet, it will print out relevant message in
 addition to requested data on the very top of the output. Thus informs the
 output data is not reliable.
-
+.\"
 .SH "DMI TYPES"
 The \s-1SMBIOS\s0 specification defines the following \s-1DMI\s0 types:
-
 .TS
 r l
 __
@@ -246,7 +247,7 @@ dmidecode --type 0,13
 dmidecode --type bios
 .IP \(bu
 dmidecode --type BIOS
-
+.\"
 .SH BINARY DUMP FILE FORMAT
 The binary dump files generated by --dump-bin and read using --from-dump
 are formatted as follows:
@@ -255,7 +256,7 @@ The SMBIOS or DMI entry point is located at offset 0x00.
 It is crafted to hard-code the table address at offset 0x20.
 .IP \(bu "\w'\(bu'u+1n"
 The DMI table is located at offset 0x20.
-
+.\"
 .SH UUID FORMAT
 There is some ambiguity about how to interpret the UUID fields prior to SMBIOS
 specification version 2.6. There was no mention of byte swapping, and RFC 4122
@@ -269,7 +270,7 @@ compatibility, it was decided to interpret the UUID fields according to RFC
 4122 (no byte swapping) when the SMBIOS version is older than 2.6, and to
 interpret the first 3 fields as little-endian (byte-swapped) when the SMBIOS
 version is 2.6 or later. The Linux kernel follows the same logic.
-
+.\"
 .SH FILES
 .I /dev/mem
 .br
@@ -278,11 +279,14 @@ version is 2.6 or later. The Linux kernel follows the same logic.
 .br
 .I /sys/firmware/dmi/tables/DMI
 (Linux only)
+.\"
 .SH BUGS
 More often than not, information contained in the \s-1DMI\s0 tables is inaccurate,
 incomplete or simply wrong.
+.\"
 .SH AUTHORS
 Alan Cox, Jean Delvare
+.\"
 .SH "SEE ALSO"
 .BR biosdecode (8),
 .BR mem (4),
diff --git a/man/ownership.8 b/man/ownership.8
index f24ef94..71ed788 100644
--- a/man/ownership.8
+++ b/man/ownership.8
@@ -1,10 +1,12 @@
 .TH OWNERSHIP 8 "February 2005" "dmidecode"
+.\"
 .SH NAME
 ownership \- Compaq ownership tag retriever
+.\"
 .SH SYNOPSIS
 .B ownership
 .RB [ OPTIONS ]
-
+.\"
 .SH DESCRIPTION
 .B ownership
 retrieves and prints the "ownership tag" that can be set on Compaq
@@ -14,7 +16,7 @@ package,
 .B ownership
 doesn't print any version information, nor labels, but only the raw
 ownership tag. This should help its integration in scripts.
-
+.\"
 .SH OPTIONS
 .TP
 .BR "-d" ", " "--dev-mem FILE"
@@ -25,11 +27,13 @@ Display usage information and exit
 .TP
 .BR "-V" ", " "--version"
 Display the version and exit
-
+.\"
 .SH FILES
 .I /dev/mem
+.\"
 .SH AUTHOR
 Jean Delvare
+.\"
 .SH "SEE ALSO"
 .BR biosdecode (8),
 .BR dmidecode (8),
diff --git a/man/vpddecode.8 b/man/vpddecode.8
index c9e4acf..1cc2b76 100644
--- a/man/vpddecode.8
+++ b/man/vpddecode.8
@@ -1,10 +1,12 @@
 .TH VPDDECODE 8 "February 2007" "dmidecode"
+.\"
 .SH NAME
 vpddecode \- \s-1VPD\s0 structure decoder
+.\"
 .SH SYNOPSIS
 .B vpddecode
 .RB [ OPTIONS ]
-
+.\"
 .SH DESCRIPTION
 .B vpddecode
 prints the "vital product data" information that can be found in almost
@@ -17,19 +19,17 @@ Box Serial Number
 Motherboard Serial Number
 .IP \(bu
 Machine Type/Model
-
 .PP
 Some systems have these additional items:
 .IP \(bu "\w'\(bu'u+1n"
 BIOS Release Date
 .IP \(bu
 Default Flash Image File Name
-
 .PP
 Note that these additional items are not documented by IBM, so this is
 guess work, and as such should not be blindly trusted. Feedback about
 the accuracy of these labels is welcome.
-
+.\"
 .SH OPTIONS
 .TP
 .BR "-d" ", " "--dev-mem FILE"
@@ -62,11 +62,13 @@ Display usage information and exit
 .TP
 .BR "-V" ", " "--version"
 Display the version and exit
-
+.\"
 .SH FILES
 .I /dev/mem
+.\"
 .SH AUTHOR
 Jean Delvare
+.\"
 .SH "SEE ALSO"
 .BR biosdecode (8),
 .BR dmidecode (8),
-- 
2.26.2

